RecoMET/METAlgorithms/src/SignAlgoResolutions.cc

```diff
@@ -385,7 +385,9 @@ void metsig::SignAlgoResolutions::initializeJetResolutions(const edm::ParameterS
     string resolutionsAlgo = iConfig.getParameter<std::string>("resolutionsAlgo");
     string resolutionsEra = iConfig.getParameter<std::string>("resolutionsEra");
 
+    [[clang::suppress]]
     string cmssw_base(std::getenv("CMSSW_BASE"));
+    [[clang::suppress]]
```

Comment:
suppress these warnings. in cmssw env both `CMSSW_BASE` and `CMSSW_RELEASE_BASE` are set

Reply:
I think this code should use `edm::FileInPath` instead.

Reply:
done